internal/2022_q4_spectaprod

```diff
@@ -36,8 +36,8 @@ During the upcoming In that time I plan to focus on the above Bounties and addit
 * [DashSee](https://trello.com/c/UALbkO5H)
 * [Decentralized Dynamic Application Infrastructure](https://trello.com/c/q16cvr5A)
 
-I am the strategist of a handful of other bounties that for various reasons are in temporarily inactive, though I have been in contact with contributors about what is needed to begin progressing again.
+I am the strategist for a handful of other bounties that, for various reasons, are temporarily inactive. I have been in contact with contributors about what is needed to begin progressing again.
 
 ### Budget Request
 
-I am requesting 750 DASH from the Incubator general budget.  This will cover the budget to fund the projects I'm administering directly, continue continuing to build a reserve fund (currently stands at 116) for launching/completing large projects in the future.
+I am requesting 600 DASH from the Incubator general budget.  This will cover the budget to fund the projects I'm administering directly, continue continuing to build a reserve fund (currently stands at 116) for launching/completing large projects in the future.
```

Comment:
Please update this:
>  600 DASH from the Incubator general budget.
to 
> 200 DASH/month for 3 months.

Dash superblocks pay out monthly so I have to request a monthly amount, repeated for 3 months in our external Incubator proposal. I want internal proposals to mirror that, so request a monthly amount for the quarter.  Oh, and again, you're not requesting from the "Incubator general fund", so just leave that part out.

Also, please fix the "continue continuing" typo.